src/SDKs/Batch/DataPlane/changelog.md

```diff
@@ -1,4 +1,9 @@
-# Azure.Batch release notes
+# Microsoft.Azure.Batch release notes
+
+## Change in 8.1.1
+### Rename Nuget package name from Azure.Batch to Microsoft.Azure.Batch
+
+# Previous Azure.Batch release notes
```

Comment:
Explain here:
"Prior to version 8.1.1, this package was named "Azure.Batch" on Nuget. The release notes below are for that package."